dotcom-rendering/index.d.ts

```diff
@@ -1173,7 +1173,7 @@ type MatchReportType = {
 interface Topic {
 	type: TopicType;
 	value: string;
-	count: number;
+	count?: number;
```

Comment:
sorry if you already explained this and I missed it but how come this now optional? Do we not always know a count?

Reply:
In DCR we are going to use the same type for both `availableTopics` & `selectedTopics` as part of this PR https://github.com/guardian/frontend/pull/25169. I think because for the `selectedTopics` is coming without a `count` 

Though, I think it's possible for us to make the count required. @joecowton1 is now working on that part in frontend to update the DotcomRenderingModel for `selectedTopic` .  So if that's all OK, I'll make the count in here as required too. But please shout if that'd be a problem

Reply:
Makes sense, was just curious to understand, thanks